src/arrayMethodJoin.js

```diff
@@ -6,6 +6,43 @@
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
```

Comment:
Add a default value for the separator
```suggestion
  [].__proto__.join2 = function(separator = ',') {
```